internal/rbd/driver/driver.go

```diff
@@ -73,11 +72,8 @@ func NewReplicationServer(c *rbd.ControllerServer) *rbd.ReplicationServer {
 
 // NewNodeServer initialize a node server for rbd CSI driver.
 func NewNodeServer(d *csicommon.CSIDriver, t string, topology map[string]string) (*rbd.NodeServer, error) {
-	mounter := mount.New("")
-
 	return &rbd.NodeServer{
 		DefaultNodeServer: csicommon.NewDefaultNodeServer(d, t, topology),
-		Mounter:           mounter,
```

Comment:
we need this change for rbd right? or else it will panic when we try to access mounter

Reply:
no, that should be set by the DefaultNodeServer... not sure why this isn't working as I expected :thinking:

Having a look now.

Reply:
Dropping the overloading `Mounter` attribute from `rbd.NodeServer` should work :crossed_fingers: